client/src/components/common/SelectMenu.scss

```diff
@@ -16,13 +16,22 @@
     .Select, .Select-control, .Select-input {
         height: 100%;
     }
-
+    .css-18ng2q5-group {
```

Comment:
I'm worried about these styles. Are we sure those names are not might be renamed at some point? An alternative is to pass in styles to the react-select component.

![Screenshot 2019-12-05 at 10 09 02](https://user-images.githubusercontent.com/31837/70221127-09ef8e00-1748-11ea-9c8f-fc7ecece2c7d.png)